Content.Server/Administration/Commands/BanCommand.cs

```diff
@@ -42,10 +66,16 @@ public async void Execute(IConsoleShell shell, string argStr, string[] args)
                 return;
             }
 
+            if (player != null && player.UserId == targetUid)
+            {
+                shell.WriteLine("You can't ban yourself!");
+                return;
+            }
+
             DateTimeOffset? expires = null;
-            if (duration > 0)
+            if (minutes > 0)
             {
-                expires = DateTimeOffset.Now + TimeSpan.FromMinutes(duration);
+                expires = DateTimeOffset.Now + TimeSpan.FromMinutes(minutes);
             }
 
             await dbMan.AddServerBanAsync(new ServerBanDef(null, targetUid, null, DateTimeOffset.Now, expires, reason, player?.UserId));
```

Comment:
I would also appreciate getting a confirm after the db write TBH.

Reply:
Will do

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3191/commits/ef317a062c4a5ee34a48d2761545d1a42c8e5f7b